src/Response/VariableResponse.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace AcquiaCloudApi\Response;
+
+/**
+ * Class VariableResponse
+ * @package AcquiaCloudApi\Response
+ */
+class VariableResponse
+{
+    public $name;
+    public $value;
+
+    /**
+     * VariableResponse constructor.
+     * @param object $variable
+     */
+    public function __construct($variable)
+    {
+        $this->name = $variable->name;
```

Comment:
Can we also pass back `links`. For the other classes I've taken an approach of converting _links to links as code sniffing tools complained about having a class property starting with an underscore so:

```
$this->links = $variable->_links;
```

Reply:
Done. Thanks